src/test/java/net/imagej/omero/it/OmeroIT.java

```diff
@@ -36,6 +36,7 @@
 
 import java.io.IOException;
 import java.net.URISyntaxException;
+import java.util.ArrayList;
```

Comment:
Why?

Reply:
Because that's where I was initially making these changes to see that they worked.